packages/flutter_tools/lib/src/commands/start.dart

```diff
@@ -38,6 +38,9 @@ abstract class StartCommandBase extends FlutterCommand {
         negatable: true,
         defaultsTo: true,
         help: 'Toggle Dart\'s checked mode.');
+    argParser.addFlag('enable-background-compilation',
```

Comment:
Is this something we want to show to users? I'm guessing this is something we'll experiment w/ for a bit, then enable by default in the engine.